### ISSUE TITLE
CENTOS - Double config values for neutron.services.loadbalancer.drivers.haproxy.plugin_driver.HaproxyOnHostPluginDriver causes neutron server to fail to start

### ISSUE THREAD

**Issue by jacobwagner** [rcbops/chef-cookbooks#803]
neutron on red hat launches with the command line

```
neutron  21530  9.9  0.2 381516 54100 ?        S    15:51   0:13 /usr/bin/python /usr/bin/neutron-server --config-file /usr/share/neutron/neutron-dist.conf --config-file /etc/neutron/neutron.conf --config-file /etc/neutron/plugin.ini --log-file /var/log/neutron/server.log
```

file /usr/share/neutron/neutron-dist.conf has

```
service_provider = LOADBALANCER:Haproxy:neutron.services.loadbalancer.drivers.haproxy.plugin_driver.HaproxyOnHostPluginDriver:default
```

overloading this in the neutron config blows up with this error

```
2014-02-20 14:27:46.873 21454 ERROR neutron.services.provider_configuration [-] Driver neutron.services.loadbalancer.drivers.haproxy.plugin_driver.HaproxyOnHostPluginDriver is not unique across providers
```

bugs
    https://bugzilla.redhat.com/show_bug.cgi?format=multiple&id=1022725
    http://openstack.redhat.com/forum/discussion/676/neutron-lbaas-doesnt-work-properly/p1

**Comment by brc** [comment]
I'm trying to think of a workaround solution for this that's not too hacky and gross.

**Comment by cloudnull** [comment]
This is resolved here : https://github.com/rcbops-cookbooks/nova-network/pull/127 : and is pending gating. 

The core issue is that RDO is having neutron point to configuration files in /usr/share/neutron and we are setting up and using the neutron config from /etc/neutron. When we add duplicate values in /etc/neutron.conf which are also in /usr/share/neutron/neutron-dist.conf oslo is appending driver values which makes neutron explode. The mentioned pull request makes sure that we only have one source of truth.

**Comment by cloudnull** [comment]
fixed and merged into 4.2.2rc: 

https://github.com/rcbops-cookbooks/nova-network/tree/v4.2.2rc
https://github.com/rcbops/chef-cookbooks/pull/820

**Comment by xeregin** [comment]
I've verified that there is no duplicate information in /usr/share/neutron/neutron-dist.conf
I've verified that there are no errors in the neutron logs:
[root@ian-compute ~]# cat /usr/share/neutron/neutron-dist.conf
[root@ian-compute ~]# cat /var/log/neutron/openvswitch-agent.log
